### ISSUE TITLE
Question about 'Joint training of the entire pipeline and Refinement-Net'

### ISSUE THREAD

**Issue by Climmy** [alex04072000/SingleHDR#32]
Hi, I encounter a problem when I run the command in ''Joint training of the entire pipeline and Refinement-Net'
![image](https://user-images.githubusercontent.com/37201120/192727284-558618d2-ea9c-4e0e-9cb2-5ff4b4fbd70d.png)
I directly load the pre-train model trained on the synthetic data. The loss doesn't diminish during the following training and the model generates blank images after several iterations.

![image](https://user-images.githubusercontent.com/37201120/192737797-5bca1917-b33b-475f-92e8-ea456626413f.png)

Some points may be helpful to figure out the cause of the problem:

1. There seems to be an inconsistency between the code and the article. According to the article, Ltv and Lp are used in the fine-tune stage.
![image](https://user-images.githubusercontent.com/37201120/192731137-895955f4-c610-428f-9b96-b3defa1f6585.png)
![image](https://user-images.githubusercontent.com/37201120/192731340-c8459352-c930-4c21-a222-2b20b829fc31.png)
However, in 'finetune_real_dataset.py',  I couldn't find Lp or Ltv:
![image](https://user-images.githubusercontent.com/37201120/192731904-be04a763-fc86-454c-bc58-d18caf33c6a0.png) 

2. In 'finetune_real_dataset.py', all weights of the four models are updated, should some of them be fixed?

Thanks a lot!
